src/pages/settings/ProfilePage.js

```diff
@@ -1,24 +1,243 @@
+            selfSelectedPronouns: initialSelfSelectedPronouns,
+            selectedTimezone: timezone.selected ?? CONST.DEFAULT_TIME_ZONE.selected,
+            isAutomaticTimezone: timezone.automatic ?? CONST.DEFAULT_TIME_ZONE.automatic,
+        };
+
+        this.pronounDropdownValues = pronounValues.map(pronoun => ({value: pronoun, label: pronoun}));
+        this.updatePersonalDetails = this.updatePersonalDetails.bind(this);
+    }
+
+    updatePersonalDetails() {
+        const {
+            firstName,
+            lastName,
+            pronouns,
+            selfSelectedPronouns,
+            selectedTimezone,
+            isAutomaticTimezone,
+        } = this.state;
+
+        setPersonalDetails({
+            firstName,
+            lastName,
+            pronouns: pronouns === CONST.PRONOUNS.SELF_SELECT ? selfSelectedPronouns : pronouns,
+            timezone: {
+                automatic: isAutomaticTimezone,
+                selected: selectedTimezone,
+            },
+        });
+    }
 
-const ProfilePage = () => (
-    <ScreenWrapper>
-        {() => (
-            <>
-                <HeaderWithCloseButton
-                    title="Profile"
-                    shouldShowBackButton
-                    onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS)}
-                    onCloseButtonPress={() => Navigation.dismissModal()}
-                />
-            </>
-        )}
-    </ScreenWrapper>
-);
+    render() {
+        return (
+            <ScreenWrapper>
+                {() => (
+                    <>
+                        <HeaderWithCloseButton
+                            title="Profile"
+                            shouldShowBackButton
+                            onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS)}
+                            onCloseButtonPress={Navigation.dismissModal}
+                        />
+                        <View style={styles.p5}>
+                            <Avatar
+                                style={[styles.avatarLarge, styles.alignSelfCenter]}
+                                source={this.props.myPersonalDetails.avatarURL}
+                            />
+                            <Text style={[styles.mt6, styles.mb6, styles.textP]}>
+                                Tell us about yourself, we would love to get to know you!
+                            </Text>
+                            <View style={[styles.flexRow, styles.mb6]}>
+                                <View style={styles.flex1}>
+                                    <Text style={[styles.mb1, styles.formLabel]}>First Name</Text>
+                                    <TextInput
+                                        style={styles.textInput}
+                                        value={this.state.firstName}
+                                        onChangeText={firstName => this.setState({firstName})}
+                                        placeholder="John"
```

Comment:
NAB since I know you didn't implement this but I think we should lighten up the placeholder value since the color is pretty much identical to when the user has entered a value 


![image](https://user-images.githubusercontent.com/16747822/111397621-f76c4000-867e-11eb-96c3-169fec53e3b1.png)

cc: @shawnborton

Reply:
Yup, placeholder color should be `themeColors.textSupporting`

Reply:
`themeColors.textSupporting` seems to be darker. here's a SS for comparison, FN is using the theme and LN is using the default color:
<img width="494" alt="Screen Shot 2021-03-17 at 6 27 56 AM" src="https://user-images.githubusercontent.com/12075600/111475165-0cca8400-86ea-11eb-91e9-8d4b0d38dcda.png">

Reply:
Got it, so it sounds like the feedback is that the color `textSupporting` is too dark as a placeholder color, right?

Maybe another option is to add a new themeColor like `themeColor.placeholderText`, and define that as `colors.gray3` in `styles/themes/default.js` - this will use a gray that is lighter than the current textSupporting color. We'd also want to make sure we update all of the textInputs to use this as a default placeholder color.

Reply:
might be better to create a `components/TextInput` with all the default styles and just use that everywhere